eslint-plugin-expensify/no-multiple-api-calls.js

```diff
@@ -0,0 +1,42 @@
+const _ = require('underscore');
+const {isInActionFile} = require('./utils');
+const message = require('./CONST').MESSAGE.NO_MULTIPLE_API_CALLS;
+
+module.exports = {
+    create(context) {
+        function checkFunctionBody(node) {
+            // This rule is scoped to the ./actions/ directory because we already have a rule
```

Comment:
I don't think we need to scope this rule to the `./actions/` directory – we can have a rule preventing any API calls outside of actions and a separate rule for preventing multiple API calls in a single function. Those don't conflict with eachother, so let's keep it simple/more flexible and get rid of the `isInActionFile` check here.

Reply:
Sounds good!